ports/gstreamer/vcpkg.json

```
@@ -424,7 +424,7 @@
     },
     "nvcodec": {
       "description": "Enable support for the NVCODEC encoders and decoders",
-      "supports": "!uwp",
+      "supports": "!(osx & arm) & !ios & !android & !emscripten",
```

The code you linked indicates 
```suggestion
      "supports": "!osx & !ios & !android & !emscripten",
```